### ISSUE TITLE
Setting normal font family + size results in error

### ISSUE THREAD

**Issue by Madoc** [alacritty/alacritty#6272]
### System

OS: macOS
Version: alacritty 0.10.1 (2844606)

### Logs

```
[0.000103416s] [ERROR] [alacritty_config_derive] Unable to load config "/Users/madoc/.config/alacritty/alacritty.yml": Config error: while parsing a block mapping, did not find expected key at line 1 column 67
```

### Description

This is my `~/.config/alacritty/alacritty.yml`:

```yaml
font:
  normal:
    family: "PragmataProMonoLiga Nerd Font"
  size: 16.0
```

It results in the above error.
Is there something wrong about the indentation?

Strangely, when I comment out the "normal" and "family" lines, it works just with the size.
Likewise, when I comment out the "size" lines, it also works just with the font family.

What am I missing? The error message implies that a required property was not found, but which one?
I also tried pasting in some other users' `alacritty.yml` that I randomly found on the internet, mostly getting the same error.

**Comment by kchibisov** [comment]
That looks correct and I don't have any issue with your config. Maybe something is wrong in the real file or so?

**Comment by Madoc** [comment]
Thanks for responding so quickly!

TL;DR: I got it to work now, and I have no idea why it works. It shouldn't.

I re-typed the whole document for scratch, to make sure that it doesn't contain any harmful invisible characters.
At first, I got the same error message again.

Then, I commented out some of the lines, added dummy lines, and so on.
In the end, I returned to the content as shown above. And then it just worked. No idea why.

For creating those files, I used two text editors: NeoVim at first, and later Sublime Text.

Could this have to do with the fact that I'm using an M1 Mac? Is there something going on with YAML parsing on those?

Anyways, as it works, I'll close the issue now.
Thanks again!

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@Madoc, I doubt that you using an M1 mac might have caused that, because Alacritty is designed to support documents encoded in the native UNIX and Windows [text/*](https://stackoverflow.com/a/25488949/9731176) formats.

If this occurs again, verify that you have not accidentally added the UTF-8 byte-order mark (BOM) or re-encoded it to something other than UTF-8. Additionally, ensure that you have utilized the correct previously described line-endings: UNIX's `\n`, or MS-DOS/Windows's `\r\n`, rather than the old macOS `\r`.
